web/src/lib/i18n/en.json

```
@@ -861,6 +884,7 @@
   "video_hover_setting": "Play video thumbnail on hover",
   "video_hover_setting_description": "Play video thumbnail when mouse is hovering over item. Even when disabled, playback can be started by hovering over the play icon.",
   "videos": "Videos",
+  "videos_count": "{count, plural, one {{count, number} Video} other {{count, number} Videos}}",
```

This isn't wrong, but I think using `#` for numbers is cleaner

```suggestion
  "videos_count": "{count, plural, one {# Video} other {# Videos}}",
```